packages/playground/windows/playground/pch.h

```diff
@@ -17,11 +17,16 @@
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Foundation.Metadata.h>
 #include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.System.h>
 
+#include <UI.Xaml.Controls.h>
```

Comment:
These are normally included in the template PCH. This one is a bit special since it needs to be able to include the MUX version instead and was previously relying on `CppWinRTIncludes` for that